src/pykiso/interfaces/dt_auxiliary.py

```diff
@@ -340,6 +340,9 @@ def _receive_message(self, timeout_in_s: float) -> None:
         """
 
 
+DTAuxiliaryInterface = AuxiliaryInterface
```

Comment:
If we have only one interface remaining, then it should also be THE AuxiliaryInterface

The assignment is to prevent a huge breaking change, but it could inherit the AuxiliaryInterface and display a DeprecationWarning at instanciation